controller_manager/src/controller_manager.cpp

```diff
@@ -773,6 +767,33 @@ controller_interface::return_type ControllerManager::configure_controller(
     // TODO(destogl): check and resort controllers in the vector
```

Comment:
Isn't this one already fixed?

Reply:
Thanks for pointing out. You are right, now it can be removed. 
Regarding your other comment, yes it was agreed in the last PR of controller sorting that this has to be done in the configure stage rather than right before activation